packaging/torcharrow/meta.yaml

```diff
@@ -11,49 +11,39 @@ requirements:
     # - { { compiler('cxx') } } # [win]
     - cmake
     - python
-    {% if osx %}
-    - boost
-    {% elif linux %}
-    - ninja-build
-    - checkinstall
-    - libssl-dev
-    - libboost-all-dev
-    - libdouble-conversion-dev
-    - libgoogle-glog-dev
-    - libgflags-dev
-    - libevent-dev
-    - libre2-dev
-    {% endif %}
   host:
     - python
     - setuptools
     {{ environ.get('CONDA_PYTORCH_BUILD_CONSTRAINT') }}
   run:
     - python
     - numpy
-    - boost
-    - libboost
     - typing_inspect
     - pandas
     - typing
     - tabulate
     - pyarrow
     - arrow
     - cffi
+    - glog 0.4.0
```

Comment:
Log from your workflow:
```
Adding .* to spec 'glog 0.4.0' to ensure satisfiability.  Please consider putting {{ var_name }}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{ pin_compatible() }} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level
WARNING:conda_build.utils:Adding .* to spec 'glog 0.4.0' to ensure satisfiability.  Please consider putting {{ var_name }}.* or some relational operator (>/</>=/<=) on this spec in meta.yaml, or if req is also a build req, using {{ pin_compatible() }} jinja2 function instead.  See https://conda.io/docs/user-guide/tasks/build-packages/variants.html#pinning-at-the-variant-level
```
I think it might be better to use do `glog==0.4.0`

Reply:
nice catch! Will update